src/main/java/ch/njol/skript/effects/EffChange.java

```diff
@@ -56,43 +56,44 @@
 @Name("Change: Set/Add/Remove/Delete/Reset")
 @Description("A very general effect that can change many <a href='../expressions'>expressions</a>. Many expressions can only be set and/or deleted, while some can have things added to or removed from them.")
 @Examples({"# set:",
-		"Set the player's display name to \"<red>%name of player%\"",
-		"set the block above the victim to lava",
-		"# add:",
-		"add 2 to the player's health # preferably use '<a href='#heal'>heal</a>' for this",
-		"add argument to {blacklist::*}",
-		"give a diamond pickaxe of efficiency 5 to the player",
-		"increase the data value of the clicked block by 1",
-		"# remove:",
-		"remove 2 pickaxes from the victim",
-		"subtract 2.5 from {points::%uuid of player%}",
-		"# remove all:",
-		"remove every iron tool from the player",
-		"remove all minecarts from {entitylist::*}",
-		"# delete:",
-		"delete the block below the player",
-		"clear drops",
-		"delete {variable}",
-		"# reset:",
-		"reset walk speed of player",
-		"reset chunk at the targeted block"})
-@Since("1.0 (set, add, remove, delete), 2.0 (remove all)")

```

Comment:
Here is another spot.
While I do agree these spaces are annoying and do need to be updated (at some point) they cause a lot of change to this PR and in turn create a lot more for us to review. 
While this clearly is simple to see what was changed, in some cases code could potentially be snuck in under our noses if we were to brush over this quickly.

Reply:
Yes, but first of all I wouldn't sneak code in and secondly, you'd have to do the check were I to make another pull request about the line changes anyways

Reply:
Moreover, I don't have access to a version of the files without line changes so I'd need to remake everything from scratch

Reply:
> Yes, but first of all I wouldn't sneak code in 

This is not an issue with you, this is an issue in general. And as a general coding practice as well as security, this should be reverted.

> Moreover, I don't have access to a version of the files without line changes so I'd need to remake everything from scratch

1) This shouldn't have been changed in the first place
2) The original version can be copy/pasted from the repo.

Reply:
Yes, the original version can be but I don't want to have to spend hours remaking everything 😕

Reply:
This still needs to be resolved. I do not want to hear the excuse "I don't want to have to spend hours remaking everything"
1) You shouldn't change things in the first place that aren't part of the PR
2) You just have to highlight this section and hit <tab>

Reply:
I literally copied that section from the skript master branch (https://raw.githubusercontent.com/SkriptLang/Skript/master/src/main/java/ch/njol/skript/effects/EffChange.java) and pasted it in EffChange and it didn't change anything so I thought it wasn't an issue. Moreover, that was not an excuse but it was the truth.